### ISSUE TITLE
Naming for sound features

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tambo#143]
This came from a discussion on slack discussing how we want to name and describe the sound features that can be enabled/disabled from the Preferences dialog. Taking excerpts from the thread related to this issue:

@terracoda said
> The main items that we have changed are that "Sound" is now "Other Sounds" and "Enhanced Sound" is now "Extra Sounds".
We also added explanatory text. Can you two have a look and let me know if these changes are good to be published with John Travoltage.
Mute Sound is changed, too. It is now "All Audio".
...the new text under Other Sounds will be "Play sonification and sound effects as you interact." And "Play an additional sound or sounds for this sim."

@Ashton-Morris said
>I feel like the explanation for Extra Sounds should be more informative about why it exists. Do you think it wold be over-kill to hint at why there are extra sounds? Like they might be useful for some learners but too much information for others?

@BLFiedler said
> ...it's a good idea. Not quite sure how to instantiate yet though. It's be nice to indicate the extra sounds are purposeful, reinforcing, but not for everyone

@terracoda said
> Play an additional sound or sounds for this sim. Extra Sounds reinforce concepts and may be helpful for some learners. How's that? "Play an additional sound or sounds to reinforce concepts. Extra Sounds may be helpful for some learners." Or "Play an additional sound or sounds to reinforce concepts. These sounds may be helpful for some and too much for others."

@BLFiedler said
> May be enough to go with "Play an additional sound or sounds that may be helpful for some learners"  Just in case the sound is not always directly tied to a concept (e.g., RaP comes ot mind where the tick mark sound is not necessarily present to reinforce proportion, but is a helpful navigation tool, though this could probably be debated)
Also, good ot avoid negative language with an indication that it's not good for everyone. Better to add uncertainty I think
If we are okay with getting a little longer.. "Play an additional sound or sounds to emphasize a part of this sim that may be helpful for some learners" Or something similar
I think that is generally true of enhanced/extra sounds, yeah? They are not deemed to be a core part of the auditory experience, so they can generally be described as "emphasis" sounds?

@terracoda said
>Love it!

@jessegreenberg said 
> Part of this recommendation is to change "Enhanced Sounds" to "Extra Sounds" at least publicly but probably across the project. I just wanted to make sure that was OK with the sound team. If so, we will want to identify what needs to change in existing code/public documentation and tools.

@jbphet said
> I'm not too fond of the "Other Sounds" label, because it begs the question "Other than what?"  It may be obvious to us that it's other than voice, but I don't know that it would be clear to most users.  I don't know if people generally think of voicing as a "sound".  Can I suggest just labeling it as "Sounds", since I think the sub-heading makes it clear what that means?
I agree with Ashton that there should probably be more of an explanation about "Extra Sounds".  If I were a user, I wouldn't find the current text to be enough of an explanation to help me decide whether I wanted them on or off.  Maybe something like "Additional sounds that may be helpful in drawing some users' attention to certain changes that occur in the sim".
Also, "Extra Sounds" makes it seem like the sounds are potentially superfluous.  Maybe "Additional Sounds" or "Extended Sounds"?

**Comment by jessegreenberg** [comment]
+1 for using "Sounds" because it stands on its own. I think "Other Sounds" could be confusing in cases where Voicing is not available and "Other Sounds" is the only item in the "Audio" tab. Also, if we go with "Sounds" maybe we can keep "Enhanced Sounds", which accurately describes it without the drawbacks of "Extra" @jbphet mentioned.

Assigning to @terracoda for next steps, but feel free to reassign to others if you want more feedback/discussion here.

**Comment by jbphet** [comment]
@terracoda - You've assigned this to me, but unfortunately I'm a little unclear as to what I need to do to move it along.  Did you want me (and @Ashton-Morris) to opine on what you've written in the previous comment?

**Comment by terracoda** [comment]
@jbphet, yes please, opine! 
Do you want to make any changesin addition to changing "Other Sounds" to "Sounds"?

Specifically, in addition to the name change do you want to change the Sounds toggle responses to include "Non-speech"?
And do you want to swap the wording of "Extra" and "Additional"?

I wanted you both to see the other content that supports the names to help you more with your opinion/decision.

**Comment by Ashton-Morris** [comment]
I also agree that **Other Sounds** can be changed to **Sounds** and I also prefer:
Additional Sounds
Play an extra sound or sounds that may be helpful for some learners.

**Comment by jbphet** [comment]
I'd say that I prefer "Sounds" over "Other Sounds".

I prefer "Additional Sounds" over "Extra Sounds".  I also don't think that it's bad to use the label in the supporting text.  So, for instance, I think it would be okay to label the checkbox with "Additional Sounds" and have the text say "Enable one or more additional sounds that may be useful for some learners".

Having said these things, I should say that I'll support whatever is decided upon, because I know that the buck has to stop somewhere, and that person has to make the final call.

On another note, in one of the comments above @terracoda said:

> We might not come up with the perfect terms/words for Version 1 of the Preferences Menu. We might have to change them, and we are in fact already planning that there will be a Version 2.

Please keep in mind that changing strings after publication means that translations of that string are no longer valid and will disappear from the sim.  That doesn't mean we can't do it, but we need to consider this and not make such changes lightly.  We currently have no mechanism in place to notify our translators that a sim that was previously fully translated has changed such that it now has some untranslated strings.

**Comment by terracoda** [comment]
@jbphet, you make a good point about translation. We are planning on doing a teacher survey to evaluate the Preferences Menu further. Is it possible to not allow translation on version 1?

**Comment by terracoda** [comment]
Hmm, given @jbphet's concern about translation, it might be beneficial for future translations to include both words, "Additional" and "extra" in our first release. If wording gets changed down the road, both words are already available in any translations that might have to be updated.

Lets try the following:

**Additional Sounds**
- _On-screen description:_ Play an extra sound or sounds that may be helpful for some learners.
- _Reading block will voice:_ "Additional Sounds, Play an extra sound or sounds that may be helpful for some learners."

**Responses when toggled:**
- Additional Sounds on.
- Additional Sounds off.

If, "Additional" sounds funny due to strange screen readers pronunciation, let's leave the main title/label the same, but switch the responses to "Extra sounds on." and "Extra sounds off." Hopefully, "Additional" will sounds nice, but sometimes screen readers have trouble with getting the stress right on 3-syllable words.

Assigning to @jessegreenberg for implementation.

**Comment by terracoda** [comment]
The Voicing Preferences Design doc has been updated, to include decisions above.
See the [Audio Tab section](https://docs.google.com/document/d/1Qu3U8IsFYn486oJUzQ71BXs_lBKZTVS5NVdLJlTK7gw/edit#heading=h.bs1d8ov39acy) in the design doc

**Comment by jbphet** [comment]
@terracoda 

> We are planning on doing a teacher survey to evaluate the Preferences Menu further. Is it possible to not allow translation on version 1?

Not really.  The only things we prevent from being translated are a11y-specific strings.  Everything else if fair game, and we'd have to add specific code to prevent this, which wouldn't be trivial.  Can we do the teacher survey with dev or RC versions instead of fully published versions?

**Comment by BLFiedler** [comment]
Remind me the timeline for this? I could probably get the survey out next week with a Dev/RC version, collect a weeks worth of info, do a quick analysis/report out the week of the 17th? Does that delay too much?

**Comment by jessegreenberg** [comment]
Voicing is still in its first dev test @BLFiedler, so I believe would have time for that before first publication and it wouldn't cause further delay.

**Comment by terracoda** [comment]
@jbphet, ok good to know, that "going live" means "getting translated"! This is all very exciting!

@BLFiedler, I was assuming that the survey you are working on is about the Preferences Menu. In the Rocks spreadsheet, the survey is spanning end of May an June.

And @jessegreenberg just commented above.

**Comment by BLFiedler** [comment]
Survey should go wherever it does the most good :) If that's pre-preferences publication, then it should be before that goes live. I'll chat with EM about it today, ping @jessegreenberg about a dev version that makes sense, and then chat with y'all about it Friday at the meetings.

**Comment by terracoda** [comment]
Two surveys have gone out. We'll have some data soon to see if we need to make any changes to the names of the sound features.

**Comment by BLFiedler** [comment]
The survey does not touch specifically on the wording of the preferences menu, but we can look at the comments to see how folks talk about the 'sounds' in comparison to the 'voicing' to see if anything sensible emerges. The comments from survey 1 & 2 are found here: https://docs.google.com/spreadsheets/d/117tjIISMwKLAwqPGqQhfuynXivbjnIlKiMeKlnYHezM/edit?usp=sharing

**Comment by BLFiedler** [comment]
Checking in on this issue - I am wrapping up updating the Teacher Tips with the latest A11y info for interactive description and updating Enhanced Sound to Extra Sound where appropriate. 

One issue we run into is that not every simulation that has an "Extra Sound" has a Preferences menu with the new vocab (Ratio and Proportion, Gravity Force Lab (regular), and Friction). I am not sure of the timeline for updates to these simulations. Should the Teacher Tips update hold off until they receive a Preferences menu update? If so, this issue may need to remain open and be updated when one of these simulations is updated so the PD materials can also be updated. This includes updates to the Sound Features videos.

Tagging @jbphet and @zepumph as knowledgable folk or responsible for above sims.

**Comment by jbphet** [comment]
@BLFiedler said:

> Tagging @jbphet...I am not sure of the timeline for updates to these simulations. Should the Teacher Tips update hold off until they receive a Preferences menu update?

I am currently not aware of any efforts to go back through existing published sims and proactively add the preferences menu.  @jessegreenberg may be in a better position to let us know whether there is such an effort.  If not, then it doesn't seem to me like the Teacher Tips should be delayed.

**Comment by jessegreenberg** [comment]
There is no plan currently to republish these sims for this. RaP and Friction may be republished sometime this year for other goals.

**Comment by BLFiedler** [comment]
Okay - well I'm not sure if we are set enough on this issue to close it, but I've updated the teacher tips and left the Enhanced Sound language for those sims without a preferences menu (listed in my last comment)

**Comment by terracoda** [comment]
What if you have 
- Extra Sounds ("Enhanced Sounds" on older sims)

**Comment by terracoda** [comment]
Just in the older ones, then that might buy you some time on updating them when and if they get updated.

**Comment by terracoda** [comment]
@BLFiedler updated the relevant teacher tips, and now Friction will soon be published with Voicing. 

Friction's current Teacher Tips mention "Enhanced Sounds"
![image](https://user-images.githubusercontent.com/1902116/182151117-ab615139-42f0-424f-8bc8-a701329d7237.png)

And the updated Preferences menu says "Extra"
<img width="391" alt="image" src="https://user-images.githubusercontent.com/1902116/182151403-bb423150-761e-4d04-b62f-e02266f0c61d.png">

Assigning to @BLFiedler. Please assign back to me if I need to make this update. I haven't updated Teaching Resources before, but that does not mean I can't do it.

**Comment by BLFiedler** [comment]
Moved this to https://github.com/phetsims/friction/issues/309

The update to the sound features and teacher tips should be standard practice for republication. I am not sure this needs to be explicitly indicated for the remaining sims, but if there is some task assignment for "Create or Update Teacher Tips" on a publication checklist then that is probably sufficient for whoever is assigned to that.

If we have that done, I think this issue can be closed.
